cli/pcluster/config/pcluster_config.py

```diff
@@ -21,6 +21,7 @@
 import configparser
 from botocore.exceptions import ClientError
 
+import pcluster
```

Comment:
do we need to import pcluster? if so can we import it locally if the error is triggered?

Reply:
I have changed the output string and the import it's not needed anymore